little_timmy/config_loader.py

```diff
@@ -27,6 +27,105 @@
 
 LOGGER = logging.getLogger("little-timmy")
 
+
+def find_and_setup_galaxy_collections(root_dir: str, skip_dirs: list[str]) -> list[str]:
+    """
+    Find all galaxy collections in the root directory and set them up for FQDN access.
+    Creates in-memory Python modules without modifying the filesystem.
+    Returns empty list (kept for compatibility).
+    """
+    import sys
+    import types
+    from glob import iglob
+    
+    # Look for galaxy.yml files which indicate a collection
+    for galaxy_file in iglob(f"{root_dir}/**/galaxy.yml", recursive=True):
```

Comment:
use the existing function for this
https://github.com/hoo29/little-timmy/blob/main/little_timmy/utils.py#L12